environment.yml

```diff
@@ -33,3 +33,4 @@ dependencies:
       - xgboost==0.80
       - nipy==0.4.2
       - pycodestyle==2.4.0
+      - nilearn==0.6.0
```

Comment:
In PR #62 of @mdiazmel regarding `t1-linear` pipeline, a bug was found on the `0.6.0` version for `resample_to_img` function. 

Since your called function is only `plotting`, I think you can safely downgrade to `0.5.0` version

Reply:
Ok